### DISCUSSION TITLE
non-enzymatic reduction

### DISCUSSION THREAD

**Discussion by gapa84** [Open-Systems-Pharmacology/Forum#1171] [Questions & Problems]
Hello. You always get a lot of help here.
Most of the drugs I want to complete the PBPK model are non-enzymatic reduction. What should I do if I want to apply that part?
For reference, I can't use PK-sim mobi yet. Is it possible to solve this without going over to mobi if possible?
It's difficult because I'm not familiar with the program yet.

I know you're busy, but I'd appreciate it if you could reply.

**Comment by StephanSchaller** [comment]
YOu can still use a "dummy" enzyme to account for non-enzymatic degradation. Depending where this happens, you can set your "fake" enzyme distribution in the specific organs/compartments
